webapp/portlet/src/main/webapp/vue-apps/attach-image/components/form/ImageInputMultiUpload.vue

```diff
@@ -77,6 +77,9 @@ export default {
         return;
       } else {
         filesArray.forEach((file) => {
+          if (!file?.type?.includes('image/')) {
```

Comment:
to use the `?.` and to ensure that it's efficient, you should use it on all elements

```suggestion
          if (!file?.type?.includes?.('image/')) {
```